pip_review/__main__.py

```diff
@@ -76,20 +78,15 @@ def parse_args():
     parser.add_argument(
         '--auto', '-a', action='store_true', default=False,
         help='Automatically install every update found')
-    parser.add_argument(
-        '--editables', '-e', action='store_true', default=False,
-        help='Also include editable packages in PyPI lookup')
-    parser.add_argument(
-        '--user', '-u', action='store_true', default=False,
-        help='Only output packages installed in user-site.')
-    parser.add_argument(
-        '--local', '-l', action='store_true', default=False,
-        help='If in a virtualenv that has global access, do not output '
-             'globally-installed packages')
-    parser.add_argument(
-        '--pre', '-p', action='store_true', default=False,
-        help='Include pre-release and development versions')
-    return parser.parse_args()
+
+
+    parsed, unknown = parser.parse_known_args() #this is an 'internal' method
+    # which returns 'parsed', the same as what parse_args() would return
+    # and 'unknown', the remainder of that
+    # the difference to parse_args() is that it does not exit when it finds redundant arguments
+    unknown = [arg for arg in unknown if arg.startswith(("-", "--"))]
```

Comment:
This was a bit overly restrictive. pip has arguments that take values, such as `--timeout`. The above line removes those values, causing `pip list` to bail out. I fixed this in 16e8ab41.